README.md

```diff
@@ -1 +1,68 @@
-Welcome to our README!  Feel free to leave some nice feedback.  :)
\ No newline at end of file
+Welcome to our README!  Feel free to leave some nice feedback.  :)
+
+# yourneighbourhoodinfo.com API
+
+## Base path!
+
+`https://yourneighbourhood.com/api`
+
+***
+
+### Introduction
+
+Welcome to our documentation for the "yourneighbourhoodinfo.com" API.
+
+This API will allow you to access, store and update information about households and their owners within your particular neighbourhood.  For example, you can:
+   - Store people, houses and addresses
+   - Look up a house, it’s address and owner
+   - Look up people in our neighbourhood within certain age brackets and with specific household size
+
+Please continue reading for more information on how to use our fabulous API!
+
+***
+
+### Data storage
+
+First of all, data stored by this API will be formatted as a JSON object in a non-relational database.
```

Comment:
We were just wondering, why did you guys decide on the non-relational database?

Reply:
Great question! 

We started out with relational but then switched due to:

- Wanting to contain all household info in same place without a need to make joins
- We thought it might be easier to query the data, especially because we also have a clear separation within each household object between people-related info and location-related info
- We could also potentially reduce the number of routes needed as all patching could go through the same path

Obviously, we could be wrong, but that's what we ended up deciding!